allensdk/brain_observatory/behavior/behavior_project_api/behavior_ophys_analysis_query_utils.py

```diff
@@ -0,0 +1,332 @@
+from pymongo import MongoClient
+import yaml
+import pandas as pd
+import numpy as np
+import json
+import os
+import glob
+import traceback
+import datetime
+
+from allensdk.brain_observatory.behavior import behavior_project_cache as bpc
```

Comment:
This comes from a development branch on @nickponvert's fork. We'll need to fold that in also